packages/core/src/lib/markdown-it/plugins/markdown-it-block-embed/services/VideoServiceBase.js

```diff
@@ -46,6 +46,14 @@ class VideoServiceBase {
     let escapedServiceName = this.env.md.utils.escapeHtml(this.name);
     containerClassNames.push(this.env.options.serviceClassPrefix + escapedServiceName);
 
+    let containerStyles = ["position: relative;", "height: 0;"];
+    // To set a specific style, add a style prop to the getDefaultOptions of any Service
+    if (this.options.style) {
```

Comment:
Hmm what's this branch for? Couldn't find any use cases for it currently (if so let's remove it), but perhaps it could be used to solve the earlier issue.

Reply:
Do you mean the first if (...) statement? If so then there is currently no use cases, but it could be used to custom style components if needed. I will see if its necessary.

Reply:
yup